.bash_profile

```diff
@@ -15,12 +15,13 @@ if [ -f "$(brew --prefix)/opt/bash-git-prompt/share/gitprompt.sh" ]; then
   __GIT_PROMPT_DIR=$(brew --prefix)/opt/bash-git-prompt/share
     source "$(brew --prefix)/opt/bash-git-prompt/share/gitprompt.sh"
 fi
-export PATH="/Users/spagolu/nvim/bin:$PATH"
-export EDITOR="/Users/spagolu/nvim/bin/nvim"
```

Comment:
This could potentially return a large number of tasks. Probably a `limit` and `offset` pagination maybe safer.